torch_audiomentations/augmentations/pitch_shift.py

```diff
@@ -0,0 +1,73 @@
+from random import choices
+import torch
+import torch.nn.functional as F
+
+from ..core.transforms_interface import BaseWaveformTransform
+from torch_pitch_shift import *
```

Comment:
I would prefer to be explicit about what is imported here

E.g.
`from torch_pitch_shift import get_fast_shifts, PitchShifter`

or something like that